tests/foreman/virtwho/cli/test_nutanix.py

```diff
@@ -171,5 +170,85 @@ def test_positive_hypervisor_id_option(
                 command, form_data['hypervisor-type'], org=default_org.label
             )
             assert get_configure_option('hypervisor_id', config_file) == value
-        target_sat.cli.VirtWhoConfig.delete({'name': virtwho_config['name']})
-        assert not target_sat.cli.VirtWhoConfig.exists(search=('name', form_data['name']))
+
+    @pytest.mark.tier2
+    @pytest.mark.parametrize('deploy_type', ['id', 'script'])
+    def test_positive_prism_central_deploy_configure_by_id_script(
+        self, default_org, form_data, target_sat, deploy_type
+    ):
+        """Verify "hammer virt-who-config deploy" on nutanix prism central mode
+
+        :id: 224ad753-6186-4b09-a72c-458839a8e412
+
+        :expectedresults:
+            Config can be created and deployed
+            The prism_central has been set in /etc/virt-who.d/vir-who.conf file
+
+        :CaseLevel: Integration
+
+        :CaseImportance: High
+        """
+        form_data['prism-flavor'] = "central"
+        virtwho_config = target_sat.cli.VirtWhoConfig.create(form_data)['general-information']
```

Comment:
Why not use the virtwho_config fixture here instead?

Reply:
> Why not use the virtwho_config fixture here instead?

@Gauravtalreja1 Because virtwho_config fixture will create prsim-flavor == element config, this case need to create prisim-flavor == central config.